flash_attn/models/llama.py

```diff
@@ -308,7 +309,28 @@ def config_from_meta_checkpoint(
         num_attention_heads=params["n_heads"],
         num_hidden_layers=params["n_layers"],
         rms_norm_eps=params["norm_eps"],
+        num_key_value_heads=params.get("n_kv_heads", None),
     )
+    multiple_of = params.get("multiple_of", 1)
+    ffn_dim_multiplier = params.get("ffn_dim_multiplier", None)
+
+    intermediate_size = 4 * config.hidden_size
+    intermediate_size = int(2 * intermediate_size / 3)
+    # custom dim factor multiplier
+    if ffn_dim_multiplier is not None:
+        intermediate_size = int(ffn_dim_multiplier * intermediate_size)
+    intermediate_size = multiple_of * ((intermediate_size + multiple_of - 1) // multiple_of)
```

Comment:
this part is rounding `intermediate_size` to nearest multiple of `multiple_of`. could we do something like `intermediate_size = multiple_of * math.ceil(intermediate_size / multiple_of)`, and add a comment?

Reply:
I've added a link to the original implementation, I believe it's good to keep the exact same code for readability?